contracts/commitment-lock/src/main.rs

```diff
@@ -250,7 +250,24 @@ fn auth() -> Result<(), Error> {
         }
 
         let raw_since_value = load_input_since(0, Source::GroupInput)?;
-        let message = load_tx_hash()?;
+        let message = {
+            let mut hasher = new_blake2b();
```

Comment:
how about using `load_transaction` to load the whole tx and remove the cell_deps to build the hash data: `load_transaction()?.raw().as_builder().cell_deps(vec![]).build().as_slice()`

Reply:
It consumes more memory (twice the size of tx), but I think that's ok.